conjur/api/api.py

```diff
@@ -79,7 +79,7 @@ def __init__(self,
 
     @property
     # pylint: disable=missing-docstring
-    def api_token(self):
+    def api_token(self) -> requests.Response:
```

Comment:
pretty sure it is str. but i could be wrong

Reply:
yep it is str

Reply:
Done